src/presentation/express-error-handler.ts

```diff
@@ -31,6 +31,12 @@ export class ExpressErrorHandler implements HttpErrorHandler {
       case ErrorCode.NOT_FOUND: {
         return HttpStatusCode.NOT_FOUND
       }
+      case ErrorCode.BAD_REQUEST: {
+        return HttpStatusCode.BAD_REQUEST
+      }
+      case ErrorCode.UNPROCESSABLE_ENTITY: {
+        return HttpStatusCode.UNPROCESSABLE_ENTITY
+      }
```

Comment:
Is it correctly understood that this models status code 422 (Unprocessable Content)?

Reply:
Yes

Reply:
Unsure if Content or Entity is the current name. The banks I've worked in used Entity, and half the internet seems to use that, but looking at a later RFC I can see that Content is used:
https://datatracker.ietf.org/doc/html/rfc9110#name-422-unprocessable-content

Reply:
LOL, even Postman thinks it is an entity, but the RFC says Content

![image](https://github.com/user-attachments/assets/c119a32b-5afa-4338-9f9a-d6f453fe6b39)